### PR TITLE
Allow manual launching of Periodic Tasks through Django Admin.

### PR DESCRIPTION
**Pull request by ginmrt**
As discussed, perhaps not the ideal place for this, but it's how I would do it.

### CODE REVIEW

File: django_celery_fulldbresult/admin.py [hunk 1]
```diff
@@ -32,5 +35,22 @@ class TaskResultMetaAdmin(admin.ModelAdmin):
...
-# Register your models here.
 admin.site.register(TaskResultMeta, TaskResultMetaAdmin)
+
+
+# Option to override PeriodicTaskAdmin
+def trigger_periodic_task(modeladmin, request, queryset):
+    """Admin Site Action for django_celery's PeriodicTask that manually
+    triggers the selected tasks.
+    """
+    retry_task(modeladmin, request, queryset)
```
Comment (bartdag) [inline django_celery_fulldbresult/admin.py:46]: this works because periodictasks has a args, kwargs and task attribute right?

**Comment by bartdag** [comment]
g2g

File: django_celery_fulldbresult/admin.py (hunk 1 above)
Comment (ginmrt) [inline django_celery_fulldbresult/admin.py:46]: Yes

https://github.com/celery/django-celery/blob/master/djcelery/models.py#L203

**Comment by bartdag** [comment]
One more thing, i never released 0.3, so you can put your change under 0.3. Thanks!

**Pull request merged**